script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_2.txt

```diff
@@ -24,7 +24,7 @@ So, I assume that she has some urgent business that she has to rely on me for.
 Yumizuka looks away awkwardly.
 It is only for a moment, and she soon returns to her normal cheerful expression.
 <Page7>
-"Habano-sensei was looking for you. He said he wanted to talk to you about your place of residence."
+"Habano-sensei was looking for you. He said he wanted to talk to you about your address."
```

Comment:
1. Place of residence feels like it has more complex associations than just address, which I think is good here
2. The next line says "what about my residence", so if you change one you gotta change both

Reply:
revert